helm/templates/deployment.yaml

```diff
@@ -60,13 +60,13 @@ spec:
             - name: VRF
               value: "{{ .Values.vrfName }}"
             - name: NEO4J_URL
-              value: "neo4j://{{ include "vkaci.fullname" . }}-neo4j:7687"
+              value: "neo4j://{{ $.Release.Name }}-neo4j:7687"
             - name: NEO4J_BROWSER_URL
               value: "{{ .Values.n4jBrowserUrl }}"
             - name: NEO4J_USER
-              value: "{{ .Values.n4jUser }}"
+              value: "neo4j"
```

Comment:
Can we please not hardcode the neo4j username into the chart. The neo4j username can be specified in a similar way to the password in the values file. A user could change it and break vkaci because it would have been hardcoded. A customer using vkaci maybe a lot more pedantic about and so they should, hence hardcoding username and passcodes specifically in an opensource project does end up breaching security vulnerabilities.

Reply:
I do not think is possible to set the username from the neo4j chart but maybe I am blind.
All I can find is setting the name of the cluster but not the username so I guess neo4j is the default user?

Reply:
Yeah possibly the native user is always going to be neo4j. Hence why I had defaulted the value in the charts values.yaml file. If a user was making their own vkaci values they wouldn't have had to set this anyway because it was defaulted. So the change to remove the n4j user variable was really not necessary.

Reply:
I can put it back and will add a comment explaining that this is just the default non-changable neo4j username as it wasn't clear for me.